tools/.golangci.yaml

```diff
@@ -87,9 +87,12 @@ linters-settings: # please keep this alphabetized
       - name: error-strings
         severity: error
         disabled: false
-      # TODO: enable the following rules
       - name: var-naming
-        disabled: true
+        disabled: false
+        arguments:
+          - []
+          - ["GRPC", "WAL"]
```

Comment:
Question: why add the two items in the block list? Please add a comment to clarify it.